composer.json

```diff
@@ -25,15 +26,25 @@
         "ext-phpiredis": "Native PHP extension for Redis connectivity. Predis will automatically utilize when available."
     },
     "require-dev": {
-        "sami/sami": "~3.3|~4.0",
-        "phpunit/phpunit": "~4.8"
+        "phpunit/phpunit": "~4.8",
+        "friendsofphp/php-cs-fixer": "~2.2.0"
```

Comment:
I'm not sure whether the installation of php-cs-fixer in project scope is a good idea, as it adds a lot of dependencies and therefore possible side-effects. Is a global installation with composer possbile in Travis?

Reply:
see comment above :D